libobs/obs-hotkey.c

```diff
@@ -1365,6 +1391,23 @@ static inline void query_hotkeys()
 		obs->hotkeys.strict_modifiers,
 	};
 	enum_bindings(query_hotkey, &param);
+#ifdef __APPLE__
+	/* Stupid hack to reset the capslock key after keys have been queried
```

Comment:
Well in some way it still is - if you press caps lock, the green LED will come on, suggesting it is in an "arrested" state just like the old hardware key was.

Pressing it again will "unlock" it, and as such only then is it a released state. It's the only key with a "hold" state, but given Apple's keyboard layout hasn't changed in decades, it's to be expected. 😏